docs/hub/hugging-face-hub.md

```diff
@@ -0,0 +1,53 @@
+---
+
+<h1>Hugging Face Hub documentation</h1>
+
+## What's the Hugging Face Hub?
+
+We are helping the community work together towards the goal of advancing Artificial Intelligence 🔥.
+
+Not one company, even the Tech Titans, will be able to “solve AI” by themselves – the only way we'll achieve this is by sharing knowledge and resources. We are building the largest open-source collection of models, datasets, and metrics on the Hugging Face Hub to democratize and advance AI for everyone 🚀. The Hugging Face Hub works as a central place where anyone can share and explore models and datasets.
+
+## What can you find on the Hub?
+
+The Hugging Face Hub hosts Git-based repositories, which are storage spaces that can contain all your files. 💾
+
+On it, you'll be able to upload and discover...
+
+* Models, *hosting the latest state-of-the-art models for NLP, vision, and audio tasks*
+* Datasets, *featuring a wide variety of data for different domains and modalities*
+* Spaces, *interactive apps for demonstrating ML models directly in your browser*
+
+Unlike other hosting solutions, the Hub offers **versioning, commit history, diffs, branches, and over a dozen library integrations**! You can learn more about the features that all repositories share in the **Repositories documentation**.
+
+## Models
+Models on the Hugging Face Hub allow for simple discovery and usage to maximize model impact. To promote responsible model usage and development, Model repos are equipped with [Model Cards](https://arxiv.org/abs/1810.03993) to inform users of each model's limitations and biases. Additional [metadata](/docs/hub/model-repos#model-card-metadata) about info such as their tasks, languages, and metrics can be included, with training metrics charts even added if the repository contains [TensorBoard traces](https://huggingface.co/models?filter=tensorboard). It's also easy to add an **inference widget** to your model, allowing anyone to play with the model directly in the browser! For production settings, an API is provided to **instantly serve your model**.
+
+To upload models to the Hub, or download models and integrate them into your work, explore the **Models documentation**. You can also choose from [**over a dozen frameworks**](/docs/hub/libraries) such as 🤗 Transformers, Asteroid, and ESPnet that support the Hugging Face Hub.
+
+## Datasets
+The Hugging Face Hub is home to over 11,000 datasets in more than 100 languages that can be used for a broad range of tasks across NLP, Computer Vision, and Audio. The Hub makes it simple to find, download, and upload datasets. Datasets are accompanied by extensive documentation in the form of **Dataset Cards** and **Dataset Preview** to let you explore the data directly in your browser. While many datasets are public, **organizations** and individuals can create private datasets to comply with licensing or privacy issues. You can learn more about **Datasets here on Hugging Face Hub documentation**.
```

Comment:
I think we have 3K datasets. You should check in incognito since you might have visibility of all datasets, including private ones.

Reply:
Oh good catch!